rustic-babel.el

```diff
@@ -26,6 +26,11 @@
   :type 'boolean
   :group 'rustic-babel)
 
+(defcustom rustic-babel-auto-wrap-main t
+  "Whether to auto wrap BODY in a \"fn main\" function call if none exists."
```

Comment:
I think you only use upper case letters when it's an argument for a function or am I missing something ?

Reply:
> Can you do me a favor and reopen the pr with a feature branch. I don't think that's the problem, but who knows ;)
> 

Yes, but another PR merge to master or some branch other?